doc/source/myrocks/zenfs.rst

```diff
@@ -91,33 +91,46 @@ Configuration
    .. sourcecode:: bash
 
       lsblk -o NAME,SIZE,ZONED
-      NAME        SIZE  ZONED
-      sda       247.9G  none
-      |-sda1    230.9G  none
-      |-sda2        1G  none
-      |-sda3       16G  none
-      sdb        15.5T  host-managed
+      NAME                    SIZE  ZONED
+      sda                       247.9G  none
+      |-sda1                    230.9G  none
+      |-sda2                        1G  none
+      |-sda3                       16G  none
+      <short_block_device_name>  15.5T  host-managed
 
 		
-#. Change the ownership of ``nvme0n2`` to the ``mysql:mysql`` user account.
+#. Change the ownership of ``<block_device_name>`` to the ``mysql:mysql`` user account.
 
    .. code-block:: bash
 	
-	  $ sudo chown mysql:mysql /dev/nvme0n2
+	  $ sudo chown mysql:mysql <block_device_name>
 		
-#. Change the permissions so that the user or owner can read and write and the MySQL group can read, in case they must take a backup, for ``nvme0n2``.
+#. Change the permissions so that the user or owner can read and write and the MySQL group can read, in case they must take a backup, for ``<block_device_name>``.
 
    .. code-block:: bash
 	
-	  $ sudo chmod 640 /dev/nvme0n2
+	  $ sudo chmod 640 <block_device_name>
 
 #. Change the scheduler  to ``mq_deadline`` with a ``udev`` rule. Create ``/etc/udev/rules.d/60-scheduler.rules`` if the file does not exist, and add the following rule:
 
    .. code-block:: text
 
-      ACTION=="add|change", KERNEL=="sd*[!0-9]|sr*", ATTR{queue/scheduler}="mq-deadline"
+      ACTION=="add|change", KERNEL=="<short_block_device_name>", ATTR{queue/scheduler}="mq-deadline"
 
-#. Create an auxiliary directory for ZenFS. For example, you could create the ``/var/lib/mysql_aux`` directory. 
+   Restart the machine to apply the rule.
+   To check if the rule was applied correctly run the following line:
```

Comment:
Not clear if you want this to be part of the process. If so, it should be two steps 
- #. Restart
- #. Verify (instead of "To check") - for parallelism in this ordered list, we start each step with a verb.